### PR TITLE
[Migration] image plugin

### PR DESCRIPTION
**Pull request by psybuzz**

### CODE REVIEW

File: tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+      )
+      .map((category: categorization_utils.RunTagCategory) =>
+        Object.assign({}, category, {
+          items: [].concat.apply([], category.items.map(explodeItem)),
+        })
+      );
+    return withSamples;
+  }
+  _tagMetadata(
+    runToTagInfo: RunTagInfo,
```
Comment (stephanwlee) [inline tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts]: nit: this is nullable

File: tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+import * as tf_backend from '../../../components/tf_backend';
+import * as categorization_utils from '../../../components/tf_categorization_utils/categorizationUtils';
+import {TfImageLoader} from './tf-image-loader';
+import {TagInfo} from '../../../components/tf_utils/utils';
+import '../../../components/tf_categorization_utils/tf-tag-filterer';
+import '../../../components/tf-dashboard-common/tf-dashboard-layout';
+import '../../../components/tf-paginated-view/tf-category-paginated-view';
+import '../../../components/tf_runs_selector/tf-runs-selector';
+
+type RunTagInfo = {
```
Comment (stephanwlee) [inline tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts]: you might be able to grab the type from category util if you don't want to repeat this type too much.

File: tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts (hunk 1 above)
Comment (psybuzz) [reply tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts]: Thanks, done.

File: tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts (hunk 2 above)
Comment (psybuzz) [reply tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts]: Done, as it's shared with other plugins like histogram.

**Pull request merged**